npm/angular/patches/@cypress+code-coverage+3.9.5.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/node_modules/@cypress/code-coverage/support-utils.js b/node_modules/@cypress/code-coverage/support-utils.js
```

Comment:
We should create an issue for updating `@cypress/code-coverage`

Reply:
Yep we have it https://github.com/cypress-io/code-coverage/issues/536